addons/website_livechat/__init__.py

```diff
@@ -1,3 +1,15 @@
 # -*- coding: utf-8 -*-
 from . import controllers
 from . import models
+
+from odoo import api, SUPERUSER_ID
+
+
+def _post_init_website_livechat(cr, registry):
+    """ Chatbot operator partners must be 'website_published' to ensure
+    their avatar is visible for end users. """
+
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    env['im_livechat.chatbot.script'].search([]).operator_partner_id.write({
+        'is_published': True
```

Comment:
Published but inactive ? Seems weird ... shouldn't we instead allow fetching their avatar soemhow ?

Reply:
Still thinking this is a bad idea. Published partners are more for reference / customer usage, not fetching avatars.